Tests/BugsnagStackframeTest.m

```diff
@@ -93,4 +95,65 @@ - (void)testInvalidFrame {
     XCTAssertNil(frame);
 }
 
+- (void)testDummyCallStackSymbols {
+    bsg_mach_headers_initialize(); // Prevent symbolication
+    
+    NSArray<BugsnagStackframe *> *stackframes = [BugsnagStackframe stackframesWithCallStackSymbols:@[]];
+    XCTAssertEqual(stackframes.count, 0);
+    
+    stackframes = [BugsnagStackframe stackframesWithCallStackSymbols:@[
+        @"",
+        @"1",
+        @"ReactNativeTest",
+        @"0x0000000000000000",
+        @"__invoking___ + 140"]];
+    XCTAssertEqual(stackframes.count, 0, @"Invalid stack frame strings should be ignored");
+    
+    stackframes = [BugsnagStackframe stackframesWithCallStackSymbols:@[
+        @"0   ReactNativeTest                     0x000000010fda7f1b RCTJSErrorFromCodeMessageAndNSError + 79",
+        @"1   ReactNativeTest                     0x000000010fd76897 __41-[RCTModuleMethod processMethodSignature]_block_invoke_2.103 + 97",
+        @"2   ReactNativeTest                     0x000000010fccd9c3 -[BenCrash asyncReject:rejecter:] + 106",
+        @"3   CoreFoundation                      0x00007fff23e44dec __invoking___ + 140",
+        @"4   CoreFoundation                      0x00007fff23e41fd1 -[NSInvocation invoke] + 321",
+        @"5   CoreFoundation                      0x00007fff23e422a4 -[NSInvocation invokeWithTarget:] + 68",
+        @"6  ReactNativeTest                     0x000000010fd76eae -[RCTModuleMethod invokeWithBridge:module:arguments:] + 578",
+        @"7 ReactNativeTest                     0x000000010fd79138 _ZN8facebook5reactL11invokeInnerEP9RCTBridgeP13RCTModuleDatajRKN5folly7dynamicE + 246"]];
+    XCTAssertEqual(stackframes.count, 8);
```

Comment:
This test should additionally assert that the frame contents are the expected values. In the case that the behavior regresses in some way in the future, it seems much more likely that a mistake would be made to accidentally change a value rather than remove or add a frame.

Reply:
Fixed in e0e1b570751eb98970fa1ebacc76b5b2e9047f00